File: tablesbook.c
//print tables book.

#include <stdio.h> 
int main()
{
	int column, row, numberOfTables, noOfRows, result = 0;
	printf("Enter a number upto which the multiplication table is to be printed: ");
	scanf("%d", &numberOfTables);
	printf("Enter the number of rows needed in each multiplication table: ");
	scanf("%d", &noOfRows);
	for(column = 1; column <= numberOfTables; column++)
	{
		printf("Multiplication table of %d: \n", column);
		for(row = 1; row <= noOfRows; row++)
		{
			result = column * row;
			printf("%d X %d = %d\n", column, row, result);
		}
	}
	return 0;
}

File: Python/tablesBook.py
# print tables book.

print('Multiplication table')
startTableNumber = int(input('Enter a number from which the multiplication table is to be printed: '))
endTableNumber = int(input('Enter a number upto which the multiplication table is to be printed: '))
countOfRows = int(input('Enter the number of rows needed in each multiplication table: '))

for tableNumberCtr in range (startTableNumber, endTableNumber+1):
    heading = "Multiplication Table of " + str(tableNumberCtr)
    print("\n" + heading)
    print("-" * len(heading))
    for countOfRowsCtr in range(1, countOfRows+1):
        print('%d * %d = %d\t' %(tableNumberCtr, countOfRowsCtr, tableNumberCtr * countOfRowsCtr))